config/env.ini

```diff
@@ -155,6 +155,8 @@ SPC_CMD_VAR_PHP_CONFIGURE_CPPFLAGS="-I${BUILD_INCLUDE_PATH}"
 SPC_CMD_VAR_PHP_CONFIGURE_LDFLAGS="-L${BUILD_LIB_PATH}"
 ; EXTRA_CFLAGS for `make` php
 SPC_CMD_VAR_PHP_MAKE_EXTRA_CFLAGS="-g -fstack-protector-strong -fpic -fpie ${SPC_DEFAULT_C_FLAGS}"
+; EXTRA_LDFLAGS_PROGRAM for `make` php
+SPC_CMD_VAR_PHP_MAKE_EXTRA_LDFLAGS_PROGRAM="-L${BUILD_LIB_PATH}"
```

Comment:
again, we should remove these from env.ini, the user doesn't gain anything by setting them

Reply:
also, configure LDFLAGS and make LDFLAGS should always be the same.

Reply:
(with the single exception of removing libraries from ./configure that would add std lib polyfills, like libpgport. 

It's actually beyond me how they thought that was a good idea in the first place.